### ISSUE TITLE
Split FSC_Encoder dataset into Training, Validation and Testing set

### ISSUE THREAD

**Issue by loryruta** [md-scanner/md-scanner#1]
We need to evaluate FSC_Encoder model on samples **outside of the training set**. 

FSC_Encoder was trained on every character of every font of the Google Fonts repository. This way, we don't have "extra" labeled sampled to validate and test our model.

An approach could be to take fonts from the Google Fonts repository and selectively assign subsets of the characters to resp. Training set, Validation set, and Testing set (percentages TBD).

**Comment by loryruta** [comment]
Having the whole dataset, we generate the training/validation/test set in the following way:

- **Test set**: we take 4 random samples (characters) for every font
- **Validation set**: we take 4 random samples (characters) for every font
- **Training set**: everything else!
